util/libcrypto.num

```diff
@@ -311,7 +311,6 @@ X509_STORE_CTX_init                     311	1_1_0	EXIST::FUNCTION:
 OCSP_RESPONSE_free                      312	1_1_0	EXIST::FUNCTION:OCSP
 ENGINE_set_DH                           313	1_1_0	EXIST::FUNCTION:ENGINE
 EVP_CIPHER_CTX_set_flags                314	1_1_0	EXIST::FUNCTION:
-err_free_strings_int                    315	1_1_0	EXIST::FUNCTION:
```

Comment:
Dropping the function altogether in a released version is bad idea. Just make it no-op.

Reply:
Note: I thought our definition of public API is defined by what's in the public header files. In that sense, this function is not public. It just happens to be exported, because it is shared by libcrypto and libssl. 

But I don't want to be dogmatic, if you prefer to keep it as a no-up to be on the safe side, I'll revert the removal.

Reply:
Our definition of our API is what's in the header files: https://www.openssl.org/policies/technical/api-compat.html

We've been more generous in the past.